src/tables.js

```diff
@@ -185,16 +185,16 @@ Flow.keyProperties.note_glyph = {
   'D3': { code: 'v70', shift_right: -0.5 },
 
   /* Triangle */
-  'T0': { code: 'v49', shift_right: -2 },
-  'T1': { code: 'v93', shift_right: 0.5 },
-  'T2': { code: 'v40', shift_right: 0.5 },
-  'T3': { code: 'v7d', shift_right: 0.5 },
+  'T0': { code: 'v49', shift_right: -2, stem_up_y_offset: -4, stem_down_y_offset: 4 },
+  'T1': { code: 'v93', shift_right: 0.5, stem_up_y_offset: -4, stem_down_y_offset: 4 },
+  'T2': { code: 'v40', shift_right: 0.5, stem_up_y_offset: -4, stem_down_y_offset: 4 },
+  'T3': { code: 'v7d', shift_right: 0.5, stem_up_y_offset: -4, stem_down_y_offset: 4 },
```

Comment:
Is there any case when the up and down offsets will be different? Why not just have one field here `y_offset` and apply it the same way to the up and down cases?

Reply:
Yes there is a case where we need two, currently only for triangle heads.
For triangles the up stem needs to be longer than the down stem.

![image](https://user-images.githubusercontent.com/1129284/44841796-19765200-ac3c-11e8-8fd2-a6b3d72fd00f.png)
![image](https://user-images.githubusercontent.com/1129284/44842120-f26c5000-ac3c-11e8-884d-51906bf7a509.png)

An option could be to move the actual glyphs so that the starting point is at the bottom of the glyph. We will then have
![image](https://user-images.githubusercontent.com/1129284/44841546-76253d00-ac3b-11e8-99e5-b96f300eb003.png)
instead of the current
![image](https://user-images.githubusercontent.com/1129284/44841572-93f2a200-ac3b-11e8-811c-c5a40bb01b65.png)

Reply:
Thanks for the explanation and the example, that makes sense.